pages/account_lists/[accountListId]/accept_invite/[inviteId].page.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { GetServerSidePropsContext } from 'next';
+import { getSession } from 'next-auth/react';
+import { getServerSideProps } from './[inviteId].page';
+
+jest.mock('next-auth/react');
```

Comment:
`__tests__/util/setup.ts` already mocks `getSession`. I would remove all the session mocking code, `import { session } from '__tests__/fixtures/session';`, and just assert that the session in the response matches that mocked session. Since we don't really care about what the session is in this code.